### ISSUE TITLE
Support for resolving reactive declarations

### ISSUE THREAD

**Issue by josdejong** [tomblachut/svelte-intellij#117]
Thanks for building this Webstorm plugin Tomasz! It works quite nicely.

It would be great if the plugin would also be able to resolve reactive declarations since this is such a core element of Svelte. I'm not sure if this is what is already discussed in #4. Currently Webstorm says "Unresolved variable or type <VARIABLE NAME>".

![afbeelding](https://user-images.githubusercontent.com/568626/81469844-987ca580-91e7-11ea-989b-cb73d107a2bf.png)

Here the code (straight from the Svelte tutorial https://svelte.dev/tutorial/reactive-declarations):

```svelte
<script>
  let count = 0;
  $: doubled = count * 2;

  function handleClick() {
    count += 1;
  }
</script>

<button on:click={handleClick}>
  Clicked {count} {count === 1 ? 'time' : 'times'}
</button>

<p>{count} doubled is {doubled}</p>
```

**Comment by tomblachut** [comment]
I know I know. This is core feature but it's really tricky to pull of because it changes core semantics of JS. 

#97 is a work in progress support of this but is blocked by #100... and updates to WebStorm.

Also, duplicate of #7

**Comment by josdejong** [comment]
👍  I can indeed imagine that this is tricky to pull off, I hope that it will work out!
